.github/actions/verify-token/src/predicate.ts

```diff
@@ -194,20 +185,19 @@ export function createPredicate(
     },
   };
 
+  // Put GitHub event payload into systemParameters.
+  // TODO(1505): Redact sensitive information.
```

Comment:
Redacting fields can be covered by #1575 

```suggestion
  // TODO(github.com/slsa-framework/slsa-github-generator/issues/1575): Redact sensitive information.
```

Reply:
Ah ok - that issue does cover both aspects of default hiding email / PII and allowing extra config.

I'll mark this one as closing 1505 then.